src/components/icons/auditor.svg

```diff
@@ -0,0 +1,9 @@
EKihVqmiHDRdu/xefxz1WF7/MQlqTSirexM5DQiGRYk1TtSx0aZmO3DkTSH1Ij+8+1xDGUi8Nj0oHd9uxWfOYRfEygV8FfQn4IRSr0hbkVUR+ZtS0M/AVrxCEKjY00STb4K3TrD/fhqxWv1DHlia2CMbni+7OmMPYW3d47dn2W+4cfcDJvQ+JLskigcp/BszGSOIBaKdqGa2qxBPFZ8CZPVhhrxdqlxn54s9IuvwUsmittkHCWNRj5pbT08vuK6JstOHshs+Va3FO8Bv/ILFXPrGAp9S6+IKM5xByRmIkfuVFcy5mYR9GcEeQ6M6H/dQUXPpBDjAwm9okFrqqq4b9fkVz/R1Wx5+wzZ1xCD2IPPUhNFZSnRoWaUEbW4KWfo6lTyPjubsamFK0IueWQGcc0YGsw9PAb5ZXZ4EaeQtr66/CRLOfV8ZbN84FVapHFdplK/EbckfGQjNVqE2S7YQuSnnjQJe3bIYNp5sjbh+9y7A99vSejpmAEpuzWKOBb6RIOR6iMNaGpBBHljISfVdi1xCNaiTLlvTiFULTwJDHDSYiVgUXAlQVGhMaIxoL5z6Qy1NM1eUgZBGvXxJ6EXrs5j/oWaytsrvzOSCPtwEIzz+LPHRozk7xyk2Fuf1nwNfiGZdnMOnslTS2jQRnBEWqqma5ukG+9q+crW9P9SQOEsTjmxKopKaOC9q4IMTE4Meoq4KhzDTWsve8dtjPa8qq2yN7ys4fv4WNkxjtvUcU3pwZzjIlsUw9iJda1tFCKIM3C++9Et+GXIl3NbAywwweYDPjdci2+3u1rlZHxx9z7+7/0DLPqjpog4ExqhNJxYq7hYlUWnZ4lGknzcMjImFcMFWF5Z5JwBEGsxtfJfqjexz/w/fY3LhNjskKjnyjZ3T0VUo9RVkt8/im2LX4jSwstgJy65oQWGy3HF5+guVbvwz1VDddJGV4/jLy+CX0/Y+t+KSADczTYgY+vHpPDOSDC+5Cmwq+drUItSSqkAh3P+CkeZj6qddpoo8RITgrOroqjqTUUFdLqmHFoGf0qka4Uvv8LBk0F4YgZWiOSO8qVJD8wI32AvDrwB8m+zz9Sf9PxuDL6OnYBZXOpjEk0+3ZOi71Q+dJaSM8xlA6t1uS2+jrmZAHeu9s2tOx6tbcPHqPbnWLvWCJ75IXZfzd43a4OeAxLpPR81TP3JrCzRMxSVG0xKCwXCKPPWyt4XYOtxDE6V9/h7u//0c2SND5cPZZJa/hpsS5b9z99hlY/O/scbZOYJDA6vSM+qeusPiFVwiPXRqPDC5R5eIBXLqA/mgYr67kosuNtfMwD7PYfiZtjb05qG+Msp5i+eMUI1EH1tfe4XjxBI888hVfILtjgkl8UeNo1qmmjg1dvx4bRtnDipRKRV4eyz+L6eIK3W3Bot5n9+tN4I+Tv/QqsF+k3i6l29zvytv/iwQXp9YMJw/ecm2k0zvFHqHUGuTc04u1wRi0p9Oe49Mb3L37ISHnsaxSPGwiQMhWS2FqnN0u8Q6FUHKxOrs29wxyMfhxCZ4z7LXIwf54TcV4kyBozvQ/eJ/l6Snt4f7ML5yHpCZzZOcr/bhyA+YuRvmE8X2i3G8q8skGPT6Dxy65rTf7tL0l8uwz6Nvfp8hFZXJ+zAt1Pp44CKXUqyiDGhAHrC+1uOQKyhg8ZnXErf/9J5rFExwePG3BdYxqKk70ECyFV6cFq+6M3G9RArE4XznPeg0ODF5kb7gZJq1Wrk3HITuA/qzAS8EWSa5MBzMeVNRMFMv7RWfWljYPZvfZFyVP2QRxWn2xrdxT3g5btoO1nt1utxwff8rm7AapZDF8iF/JYkiefrfouu20EpiVfM5Gn8n54m3iUgBVGAbi4T5xrx2r+60uGTMTNlvC8X3q5FI8GOtHSgA6pFm8LYzxt7EuczxWdv4eg84eSkpB2I9CdXpGvnb7/KkbXGMgPPeMfZd7nQV0GWUQGEQYRtAVwLkUVKXHnjpbg6CZOAPh6d0P+Pyz79F1K1Pn7hAVcoGqaSGrJamxGXmm7dQD0pZ6MybMjhT0852BamfLgjwK/EoQCYfAy8AYJB4x6jnJFJJ3pjISqnk1pdupUroomf1nmQ4jLVh31L7vrHF27hk0sx3OuHf/I+jORnr4RIdS240Oupzz2KdFijrMZYEyJSI1qqUCPikgLADMyP4e0jYGbAefFFOj6wmenis3fJJhu7LuAY7uAx/lc2aLjmLVceHklPzxZ4xhr/IoPWgev4Q2jQ+5MdWsLu2KF53d/irSrifTa6ZTHX92mhnU1w9j+CS31yLK7ev/xs3b79r1jhqy1BBbJZClP5ObX7MxGDqZJINnws7Je2/oNCuFmF2qIr8WksRvBAm/OAZSZW5bWQLbCo8sp6dMjcOzkxJKR1Ob0lNacojHCV1azir4T1Y3OTm5SpJCSZIJfH7C2RdYB6vAsjGoltGw5oqDH8fUeiyXNrTK2Mo0KwyK9hlZtoS6otzzEngeVXrXE2bKbnLE/j/ITfD64lEy+3f3LRoCrDbkqze/8L7yjfHZp+DigV0/ZdSMbbdSoWeSz4K9g2Z6Nfb5oJk++9NBqJp9KE5x7sxm7zdHXP3snzlZ3TL8U5yKzMh+CZZ8qGODSPSuZpkdVJVIvXfOKD10Sg/C+ZrMHl9NiPwEyk08xDiRlkz6GXk0GQkVLNFcepC4Wx3EqfXBsx0yM9e1SBNbwkF77p58Sre+QytG8dmhQJUTt+EaZPEYlA6oWqu0iI72JUNwDzCP9qv4QkpWY2WUeFfbsNpurUouVZSmRtu+pzs5RTebc47NfC/bv5OFN7fI5NzRD17u6T32rv7TG6xWK6q2ndqUCayHnv7iPlw4QK/dQqUapZ6qefaDyKh2syo9nv/FVbD6NE7XFsNcDfu5BTed7t15j+s3/pO9y69TykinDeTptlLWGTash43NPCmaRgTUoh3lfkNA1CaYqujUp2e2Lgo3/g/gaJXc7UYL9gAAAABJRU5ErkJggg=="/>
```

Comment:
Can this be an actual SVG, rather than an embedded PNG?

Reply:
the icon is dummy